slam-libraries/Makefile

```diff
@@ -8,7 +8,9 @@ bufsetup:
 	ln -sf `which grpc_cpp_plugin` grpc/bin/protoc-gen-grpc-cpp
 	pkg-config openssl || export PKG_CONFIG_PATH=$$PKG_CONFIG_PATH:`find \`which brew > /dev/null && brew --prefix\` -name openssl.pc | head -n1 | xargs dirname`
 
-buf: bufsetup
+buf: bufsetup bufcompile
+
+bufcompile:
```

Comment:
I ran into an issue where if I forgot to run `make buf` before `make buildorb` the only way I could get the compiled proto classes was to also reinstall buf (which is not necessary). As a result I've split up the part that does code gen into its own make command called `bufcompile` so that it is not required to install buf every time you want to do proto code gen.